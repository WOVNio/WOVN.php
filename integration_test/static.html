<html>
  <head></head>
  <body>
    <h1>Static Content</h1>
  </body>
</html>
