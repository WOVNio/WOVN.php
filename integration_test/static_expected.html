<html>
  <head><link rel="alternate" hreflang="ja" href="http://localhost/static.html?a=b&amp;wovn=ja"><link rel="alternate" hreflang="fr" href="http://localhost/static.html?a=b&amp;wovn=fr"><link rel="alternate" hreflang="bg" href="http://localhost/static.html?a=b&amp;wovn=bg"><link rel="alternate" hreflang="en" href="http://localhost/static.html?a=b"><script src="//j.wovn.io/1" data-wovnio="key=Tek3n&amp;backend=true&amp;currentLang=en&amp;defaultLang=en&amp;urlPattern=query&amp;langCodeAliases=[]&amp;langParamName=wovn&amp;version=WOVN.php"  async></script></head>
  <body>
    <h1>Static Content</h1>
  </body>
</html>
